src/gdal-2-openjpeg2.1.0rev27538.patch

```diff
@@ -0,0 +1,247 @@
+diff -bur gdal-1.11.1.orig/configure gdal-1.11.1/configure
+--- gdal-1.11.1.orig/configure	2014-10-24 10:57:41.487491392 +0530
++++ gdal-1.11.1/configure	2014-10-24 10:58:16.354947432 +0530
```

Comment:
@TimothyGu, could you tell about autoreconf -fi, using that I dont need to modify configure script is that right?